lib/utils.js

```diff
@@ -22,6 +22,37 @@ export function calculateSlippageAmount(value) {
   return value.mul(SLIPPAGE_PERCENTAGE).div(100)
 }
 
+export function useCheckEmailForAddress() {
+  const [userExists, setUserExists] = useState(false)
+  const { account: address } = useWeb3Connect() || ''
+  return useCallback(async () => {
+    try {
+      const response = await fetch(`${env('BACKEND_URL')}/user/${address}`)
+      if (!response.ok) {
+        throw new Error(
+          `Received a wrong status when trying to fetch if user exists: ${response.status}.`
+        )
+      }
+      console.log(response)
```

Comment:
We should remove this console.log().

Reply:
Ah yes, this is while I test for the email API. :)